src/drivers/intel/ssp/ssp.c

```diff
@@ -756,6 +757,14 @@ static int ssp_set_config_tplg(struct dai *dai, struct ipc_config_dai *common_co
 		}
 		break;
 	case SOF_DAI_CONFIG_FLAGS_HW_FREE:
+		/* disable SSP port if no users */
+		if (ssp->state[SOF_IPC_STREAM_CAPTURE] != COMP_STATE_PREPARE ||
+		    ssp->state[SOF_IPC_STREAM_PLAYBACK] != COMP_STATE_PREPARE) {
+			dai_info(dai, "ssp_set_config(): hw_free stage: ignore since there is still user",
+				 dai->index);
```

Comment:
`dai_info()` has one parameter but the format string has no specifier. Found by chance by the Zephyr build which has a different logging framework. Fix submitted in #4727